src/main/java/EventCommand.java

```diff
@@ -0,0 +1,20 @@
+import java.time.LocalDateTime;
+
+public class EventCommand extends Command {
+    private String event;
+    private LocalDateTime start;
+    private LocalDateTime end;
+
+    public EventCommand(String event, LocalDateTime start, LocalDateTime end) {
+        this.event = event;
```

Comment:
Nice use of start and end for event and due for deadline to differentiate the usage! 👍